lib/model/yake_keywords.py

```diff
@@ -36,6 +37,8 @@ def normalize_special_characters(self, text):
             text = text.replace(k, v)
         return text
 
+    def run_chinese_segmentation_with_jieba(self, text):
+        return " ".join(list(jieba.cut_for_search(text)))
```

Comment:
add newline after function please

Reply:
I don't see a need for this to be a separate function given it's one line and only invoked in one place. I'd just include `text = " ".join(list(jieba.cut_for_search(text)))` right in the `run_yake` function

Reply:
I wanted to put it in a separate function so I can write tests separately for it. namely `test_run_chinese_segmentation_with_jieba` As this step includes downloading and loading a model. Is that the clean way to do so?